### PR TITLE
Issue 294

### PR DESCRIPTION
**Pull request by strawberries73**
Good afternoon.  I worked on this before sending you the issue.  I am done.  I am new at this, so please let me know how I did.

### CODE REVIEW

File: animations/harvestflare.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/**
+ * /* By: https://github.com/strawberries73
+ * @format
+ */
+
```
Comment (NiallEccles) [inline animations/harvestflare.css:5]: Just need to add the `Staatliches` font import here `@import url('https://fonts.googleapis.com/css2?family=Staatliches&display=swap');`

File: animations/harvestflare.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+		color: #806914;
+    }
+    25% {
+		text-shadow: 1.5px 1.5px 0 rgb(228, 124, 13),
+			-1.5px -1.5px 0 rgb(185, 172, 23);
+	}
+}
+
+.harvestflare {
+	font-family: 'Staatliches', cursive;
```
Comment (NiallEccles) [inline animations/harvestflare.css:26]: This font won't display unless you import that specific font or the user has it installed. Easy fix!

**Review by NiallEccles — CHANGES_REQUESTED** [review]
Thanks for submitting this! I just have one change for you to make. It is to add in the import statement for the font you are using.

File: animations/harvestflare.css (hunk 1 above)
Comment (strawberries73) [reply animations/harvestflare.css:5]: Ok, I shall do that.  Thank you!

Comment (strawberries73) [reply animations/harvestflare.css:5]: Hello, I have made the changes. Is there anything else I can do?

Comment (NiallEccles) [reply animations/harvestflare.css:5]: Nope! That should be it. Have you made the changes to this request? I can't seem to see it right now. If you can update this request with that change I can merge!

Comment (strawberries73) [reply animations/harvestflare.css:7]: I think I got it this time.  I am still getting acquainted with GitHub.

Comment (NiallEccles) [reply animations/harvestflare.css:7]: No problem! Glad to have your contribution!

**Pull request merged**